### DISCUSSION TITLE
Windows Authentication

### DISCUSSION THREAD

**Discussion by Agallo10** [sequelize/sequelize#15183] [Help & Questions]
Does anyone knows if sequelize supports Windows Authentication. please help me.

**Comment by fzn0x** [comment]
Can you elaborate about that? Sequelize is a NodeJS orm, not for some low level stuff.

**Reply by Agallo10** [reply]
Hi, I'm trying to connect to a mssql database which uses windows authentication. with nodeJs - sequilize. an you help me? please.

**Reply by WikiRik** [reply]
You will probably need to use `dialectOptions` for this so it will be passed directly to `tedious`. See https://sequelize.org/docs/v6/other-topics/dialect-specific-things/#mssql for more information about this

**Reply by Agallo10** [reply]
Thanks, i find the solution here: 
https://github.com/vishwasjois/sequelize/blob/988e754c6eef323b1a9dc11f5bee3fb535579da8/docs/upgrade-to-v5.md#dialect-specific
